DeprecationWarning: "--compilers" will be removed in a future version of Mocha;

- mocha v4.0.1
- power-assert v1.4.4
- espower-typescript v8.1.2

I got a following warning after running mocha command.

```
> mocha --compilers ts:espower-typescript/guess



DeprecationWarning: "--compilers" will be removed in a future version of Mocha; see https://git.io/vdcSr for more info

  ✓ example test

  1 passing (6ms)
```

after see https://git.io/vdcSr , I've changed `--compilers` to `--require`, but no testcase found. 😢 

```
> mocha --require espower-typescript

No test files found
npm ERR! Test failed.  See above for more details.
```

Is there a solution?

--- a/README.md
+++ b/README.md
@@ -41,7 +41,12 @@ describe('Array#join', () => {
 ```
 
 ```console
-$ ./node_modules/.bin/mocha --compilers ts:espower-typescript/guess test/**/*.ts
+// for mocha version >= 4
+$ ./node_modules/.bin/mocha --require espower-typescript/guess "test/**/*.ts"
+
+
+// for mocha version < 4 (deprecated)
+$ ./node_modules/.bin/mocha --compilers ts:espower-typescript/guess "test/**/*.ts"
 ```
 
 ### If your tests are not in `test` directory
@@ -60,10 +65,14 @@ You can set test directory in your `package.json`
 }
 ```
 
-Then, run mocha with `--compilers ts:espower-typescript/guess`
+Then, run mocha with `--require espower-typescript/guess`
 
 ```console
-$ ./node_modules/.bin/mocha --compilers ts:espower-typescript/guess spec/**/*.ts
+// for mocha version >= 4
+$ ./node_modules/.bin/mocha --require espower-typescript/guess "spec/**/*.ts"
+
+// for mocha version < 4 (deprecated)
+$ ./node_modules/.bin/mocha --compilers ts:espower-typescript/guess "spec/**/*.ts"
 ```
 
 Note: `'espower-typescript/guess'` is inspired by [intelli-espower-loader](https://github.com/azu/intelli-espower-loader)

